src/main/java/com/jalasoft/jfc/model/PdfConverter.java

```diff
@@ -36,11 +36,11 @@ public Boolean convert(PdfParam pdf){
 
         Boolean isConverted = false;
         try {
-            if (pdf.getPathInput() == null || pdf.getPathOuput() == null) {
+            if (pdf.getInputPathFile() == null || pdf.getOutputPathFile() == null) {
                 throw new IllegalArgumentException();
             }
             PDDocument documenttoimage = PDDocument.load(
```

Comment:
please use camel case for variable names "documenttoimage" for example **documentToImage**

Reply:
ok, I changed the expection by IO